Sources/SwiftProtobuf/GetPathDecoder.swift

```diff
@@ -0,0 +1,324 @@
+// Sources/SwiftProtobuf/GetPathDecoder.swift - Path decoder (Getter)
+//
+// Copyright (c) 2014 - 2023 Apple Inc. and the project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See LICENSE.txt for license information:
+// https://github.com/apple/swift-protobuf/blob/main/LICENSE.txt
+//
+// -----------------------------------------------------------------------------
+///
+/// Decoder which captures value of a field by its path.
+///
+// -----------------------------------------------------------------------------
+
+import Foundation
+
+struct GetPathDecoder<T: Message>: Decoder {
```

Comment:
Generally could use a comment to say what its used for, etc.

Reply:
Done